service/interfaces.go

```diff
@@ -31,3 +33,9 @@ type HealthChecker interface {
 	Stop()
 	AddCheck(name string, checker healthcheck.Checker) (err error)
 }
+
+// MongoServer represents all the required methods from mongo DB
+type MongoServer interface {
```

Comment:
Consider naming this interface more in line with its functional purpose - in this case something like `AreaStore`?